test/testutil/storage/in_memory_storage.cpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "in_memory_storage.hpp"
+
+#include "in_memory_batch.hpp"
+
+using kagome::common::Buffer;
+
+namespace test{
+
+  outcome::result<Buffer> InMemoryStorage::get(const Buffer &key) const {
```

Comment:
better move it to `core/storage/inmem`

Reply:
I did firstly, but then realised that there's still no need for them there. Especially now that PolkadotTrie is separate and it's easy to create an in-memory trie without tricks with a storage.

Reply:
this class can be used in other things, not only in trie db. 
for examlpe when you need in memory kv db

Reply:
Okay, if you think it may be useful.